net-dialup/linux-atm/files/linux-atm-2.5.2-c99-musl.patch

```diff
@@ -0,0 +1,115 @@
+diff -pur a/configure.in b/configure.in
```

Comment:
I know upstream is dead, but can you report this upstream and include the patch there anyway? I think it's useful in this case because a lot of people still have linux-atm packaged so it gives a place to coalesece.

Reply:
I can't, unable to register to Sourceforge.
Please forward it for me if possible.

Reply:
OK.

Reply:
I have a sourceforge account, but please add some into at the top of the patches, at least the author and a link to the gentoo bug, then I can upload them.